packages/getting-started/src/browser/getting-started-preferences.ts

```diff
@@ -22,21 +22,32 @@ import {
     PreferenceSchema,
     PreferenceContribution
 } from '@theia/core/lib/browser/preferences';
+import { FrontendApplicationConfigProvider } from '@theia/core/lib/browser/frontend-application-config-provider';
 import { nls } from '@theia/core/lib/common/nls';
 
 export const GettingStartedPreferenceSchema: PreferenceSchema = {
     'type': 'object',
     properties: {
-        'welcome.alwaysShowWelcomePage': {
-            type: 'boolean',
-            description: nls.localizeByDefault('Show welcome page on startup'),
-            default: true
-        }
+        'welcome.startupEditor': {
```

Comment:
We should use the same preference `id` present in VS Code in order to support their preference natively (reading from `.vscode/settings.json`):

```suggestion
        'workbench.startupEditor': {
```